### ISSUE TITLE
Integrate with WebsiteLink

### ISSUE THREAD

**Issue by AaronMcHale** [NewEdenMC/PlayTimer#8]
https://neweden.co/api/doesUserExist method is now deprecated and will be removed, integrate with WebsiteLink plugin instead, see diff below for reference:

```
diff --git a/src/co/neweden/playtimer/PlayTimer.java b/src/co/neweden/playtimer/PlayTimer.java
index aed190f..b94cb40 100644
--- a/src/co/neweden/playtimer/PlayTimer.java
+++ b/src/co/neweden/playtimer/PlayTimer.java
@@ -18,6 +18,9 @@ import org.bukkit.plugin.RegisteredServiceProvider;
 import org.bukkit.plugin.java.JavaPlugin;
 import org.bukkit.scheduler.BukkitScheduler;

+import co.neweden.websitelink.User;
+import co.neweden.websitelink.jsonstorage.UserObject;
+
 public class PlayTimer extends JavaPlugin implements Listener {

        public static ArrayList<Player> verifiedUsers = new ArrayList<Player>();
@@ -94,7 +97,8 @@ public class PlayTimer extends JavaPlugin implements Listener {

                                // Checks if the user exists on the forums and gives a
                                // registration link if they don't
-                               if (HTTPRequest.forumUserExists(userName) == false) {
+                               UserObject apiObject = User.getUser(player);
+                               if (apiObject != null && apiObject.userExists == false) {
                                        player.sendMessage(Util
                                                        .formatString("&f[&8PlayTimer&f]: "
                                                                        + "&6Congratulations! You have played for atleast 12 hours, this means you are eligible to be promoted to from Explorer to Builder, all you need to do now is to register on our forum, go to http://pngn.co/q to get started. Remember to register using your Minecraft username, otherwise you won't get promote"));
```

**Comment by AaronMcHale** [comment]
Updated diff code above to comply with changes in https://github.com/NewEdenMC/WebsiteLink/commit/597dcb133cf6c25efedef90d8548b5c2f6a1f24a.  Also made it check for a null response as WebsiteLink may return null if an error was thrown.

**Comment by drisc** [comment]
Issue closed with d16930a1814623f86c11fa25056df3f659e4d4c6.
